curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/pleasure-to-meet-you.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 65528a7c39eb38f51e0e6f7f
+title: Pleasure to Meet You
+challengeType: 19
+dashedName: pleasure-to-meet-you
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+Expressions like `It's a pleasure to meet you` are commonly used when meeting someone for the first time to show politeness and appreciation when meeting someone for the first time.
+
+# --question--
+
+## --text--
+
+What does Tom say to express he is happy to meet Maria?
```

Comment:
```suggestion
What does Tom say to indicate he is happy to meet Maria?
```
"express" on its own like that seems a little weird, but "show", "indicate", "demonstrate", etc. work well.